tests/unit_tests/test_wave_01.py

```diff
@@ -40,16 +40,14 @@ def test_removing_from_inventory_returns_item():
     assert item not in vendor.inventory
     assert result == item
 
-@pytest.mark.skip
+# @pytest.mark.skip
 def test_removing_not_found_is_false():
     item = "item to remove"
     vendor = Vendor(
         inventory=["a", "b", "c"]
     )
 
     result = vendor.remove(item)
-
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert len(vendor.inventory) == 3
+    assert result is False
+    assert item not in vendor.inventory
```

Comment:
Nice assertions, we definitely want to confirm our return value and the list length. Something to consider: checking `item not in vendor.inventory` doesn't guarantee that the list is unchanged, just that `item` isn't present in the list. If we want to confirm that there weren't side affects or changes to the input that we didn't intend, we could check for the presence of the 3 items we expect. 
- What might we assert about the vendor's inventory to confirm that its contents didn’t change?